src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/renametool/SpecialRenameUserPage.java

```diff
@@ -41,7 +41,7 @@ public String encodeToURL(String toEncode) {
     try {
       encodedString = URLEncoder.encode(toEncode, "UTF-8").replace("+", "_");
     } catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)